lib/wallaby/experimental/selenium.ex

```diff
@@ -0,0 +1,56 @@
+defmodule Wallaby.Experimental.Selenium do
+  @moduledoc false
+
+  @behaviour Wallaby.Driver
+
+  alias Wallaby.Session
+
+  @type start_session_opts ::
+    {:remote_url, String.t} |
+    {:capabilities, map} |
+    {:create_session_fn, ((String.t, map) -> {:ok, %{}})}
+
+  @spec start_session([start_session_opts]) :: {:ok, Session.t}
+  def start_session(opts \\ []) do
+    base_url = Keyword.get(opts, :remote_url, "http://localhost:4444/wd/hub/")
+    capabilities = Keyword.get(opts, :capabilities, %{})
+    create_session_fn = Keyword.get(opts, :create_session_fn,
+                                    &Wallaby.Phantom.Driver.create_session/2)
```

Comment:
Ideally they could live in their own module but for now lets separate them. Once we have support for 3 drivers (phantomjs, selenium, chromedriver, etc.) then we can see if they share enough functionality to merge the driver implementations.